tests/spec/create.spec.js

```diff
@@ -17,52 +17,98 @@
  under the License.
  */
 
-const spec = __dirname;
-const path = require('path');
 const fs = require('fs-extra');
-const { superspawn } = require('cordova-common');
+const os = require('os');
+const path = require('path');
+const xcode = require('xcode');
+const create = require('../../bin/lib/create');
 
-const cordova_bin = path.join(spec, '../..', 'bin');
-const tmp = require('tmp').dirSync().name;
+const makeTempDir = () => path.join(os.tmpdir(), `cordova-ios-create-test-${Date.now()}`);
```

Comment:
IIRC there are occasional issues with `os.tmpdir()` being a symlink on macOS. The form we use in lib and android is:
```js
function makeTempDir () {
    const tmpDirTemplate = path.join(os.tmpdir(), 'cordova-android-test-');
    return fs.realpathSync(fs.mkdtempSync(tmpDirTemplate));
}


```

Reply:
I know this was used in Android as this is where my script came from, but I removed part of it because it also created the folder which should not happen. The create script requires the folder to not exist.

I could remove `fs.mkdtempSync` and write it like this:

```javascript
const makeTempDir = () => path.join(
    fs.realpathSync(os.tmpdir()),
    `cordova-ios-create-test-${Date.now()}`
);
```

Reply:
I see. Didn't get the must not exist part. Good solution!